ingestion_server/ingestion_server/indices/mappings/audio.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I think it makes sense to add `integer` mappings only if we plan to add some filtering/search for these fields. Do [we plan]/[you think we should] to add `bit_rate` or `sample_rate` queries?

Reply:
Dropped fields in 2045eee.